### PR TITLE
màj config.h pour cran pour dimming led, et problème de modtap

### PR DESCRIPTION
**Pull request by niolang**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lily58/keymaps/lily58l/keymap.c [hunk 1]
```diff
@@ -271,20 +271,36 @@ void render_status_main(void) {
     render_keylogger_status();
 }
 
-bool oled_task_user(void) {
+void oled_task_user(void) {
```
Comment (drashna) [inline keyboards/lily58/keymaps/lily58l/keymap.c:274]: Revert: 
```suggestion
bool oled_task_user(void) {
```

File: keyboards/lily58/keymaps/lily58l/keymap.c [hunk 2]
```diff
@@ -271,20 +271,36 @@ void render_status_main(void) {
...
   update_log();
   if (is_keyboard_master()) {
     render_status_main();  // Renders the current keyboard state (layer, lock, caps, scroll, etc)
   } else {
     render_lily58_logo();
   }
```
Comment (drashna) [inline keyboards/lily58/keymaps/lily58l/keymap.c:280]: revert: 
```suggestion
  }
  return false;
```

**Review by drashna — CHANGES_REQUESTED** [review]
Most of these changes should be moved to a different keymap, rather than modifying this one.

**Comment by precondition** [comment]
For a nebulous reason, GitHub doesn't let me review this PR (although it works with any other PR) so I'll post my suggestions and my comments here.

----

```diff
- Copyleft 2021 Niolang - personnal adaptation for b├ępo
+ Copyleft 2021 Niolang - personal adaptation for bépo
```

---

I'd recommend importing the bépo aliases (`BP_xxx`) over the default US QWERTY aliases (`KC_xxx`) with `#define "keymap_bepo.h"`. It would make a lot more sense and improve the readability. As it stands, your ASCII art is totally disconnected from what the code may lead to believe. More info on language-specific keycodes can be found here : https://docs.qmk.fm/#/reference_keymap_extras?id=language-specific-keycodes. An example of a keymap using Bépo keycodes can be found here: https://github.com/qmk/qmk_firmware/blob/50d4dfd2e65904234d7ee138b57eccd4b6664e1f/keyboards/ergodox_ez/keymaps/bepo_tm_style/keymap.c

---

```diff
    //return true;

//intercepting hold/tap to change hold function
	switch (keycode) {
        case MT(KC_CAPS, KC_ESC):
            if (!record->tap.count && record->event.pressed) {
-                tap_code16(KC_CAPS); // Intercept hold function to send Ctrl-X
+               tap_code(KC_CAPS); // Intercept hold function to send Caps Lock
                return false;
            }
            return true;             // Return true for normal processing of tap keycode
-        case MT(KC_NUHS, KC_H):
+        case MT(BP_CCED, KC_H):
            if (!record->tap.count && record->event.pressed) {
-                tap_code16(KC_NUHS); // Intercept hold function to send Ctrl-X
+                tap_code(BP_CCED); // Intercept hold function to send ç
                return false;
            }
            return true;             // Return true for normal processing of tap keycode			
    }
```
The first argument to `MT(mod, kc)` should be `MOD_xxx`, not a keycode but I can see why you'd want to do that

PS: N'oublie pas de traduire le titre de tes PRs en anglais.

**Comment by niolang** [comment]
Hello, this PR was actually a mistake due to noobiness using git, and lack of documentation reading...
i deleted this fork and created a new one, in which i'll take all your advices in account (regarding bépo aliases, and creating a new keymap rather than modifiying someone else's)

thanks a lot for your time

**Pull request closed without merging**